app/assets/stylesheets/components/_typography.scss

```diff
@@ -14,6 +14,9 @@ body { -webkit-font-smoothing: antialiased; }
 .fs-12p { font-size: 12px; }
 .fs-20p { font-size: 20px; }
 
+.color-blue { color: $blue; }
+.color-navy { color: $navy; }
+
```

Comment:
These color classes already exist with basscss, you can access them with `.blue` and `.navy`. We have all of these http://basscss.com/v7/docs/colors/ defined in `stylesheets/variables/colors`.